### PR TITLE
upd to the relevant version of peer deps

### PR DESCRIPTION
**Pull request by qetr1ck-op**
Tested as well, so don't see a reason to update.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
 	"name": "simple-progress-webpack-plugin",
 	"description": "A simple progress plugin for Webpack, coming with four different logging output formats.",
-	"version": "1.1.1",
+	"version": "1.1.2",
```
Comment (dominique-mueller) [inline package.json]: Please don't increment version numbers in PRs, the automatic release process will do it for us :)

**Comment by qetr1ck-op** [comment]
@dominique-mueller fixed

**Comment by dominique-mueller** [comment]
Quick question: With which Webpack versions did you test it? Originally this was designed to work with Webpack 2.x (which I think we should keep in the range).

**Comment by qetr1ck-op** [comment]
In my case, the version is `^3.6.0`. Understandable, that you've created the plugin when `3.x` version wasn't even available. 
By the way, the plugin is awesome, thanks for your job.

**Comment by dominique-mueller** [comment]
Thanks so much :)

Ah ok, then I'd suggest we keep the 2.x versions in the range, right? Would you mind updating your PR one more time?

**Comment by qetr1ck-op** [comment]
Done

**Comment by dominique-mueller** [comment]
Thanks for the contribution, just released **[version 1.1.2](https://github.com/dominique-mueller/simple-progress-webpack-plugin/releases/tag/1.1.2)** which includes the changes of this PR. Have fun!

**Pull request merged**